### PR TITLE
exploration plutonia experience - hold that thought actually this branch got fucked up

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
image
## Why It's Good For The Game
if the pathfinder has a funny burst rifle there should be a mercenary equivalent
## Changelog
:cl:
add: Scarborough Arms has invested in higher-velocity 10mm rounds and given them to special units.
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
new ballistic mobs: i sleep

**Comment by Hatterhat** [comment]
they're hitscanners @lolman360

**Comment by lolman360** [comment]
yeha but haha funny 0.3 brutemod

anwyays add like, grenadier hivebots with projectile grenades
i want to see limbs get ripped off

**Comment by CharlesWedge** [comment]
Dang it Hatter  I was working on a Merc PR too

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -217,6 +217,10 @@
 	SA_bonus_damage = 35 // 50 total on animals.
 	SA_vulnerability = SA_ANIMAL
 
+/obj/item/projectile/bullet/rifle/a545/suppressor
+	armor_penetration = 55
+	hitscan = TRUE
```
Comment (Zandario) [inline code/modules/projectiles/projectile/bullets.dm:222]: No reason for this to be hitscan.

**Review by Zandario — CHANGES_REQUESTED** [review]
Please balance this, no reason for smg-like weapon to be firing like a hitscan rifle

**Comment by Hatterhat** [comment]
pathfinders get a quirky burst rifle
this is my counterpoint

**Comment by CharlesWedge** [comment]
That is not a good counterpoint Hatter, there is no reason to add hitscanners in. I have a merc PR in the works that will more than solve their current lack of firepower in  way that is fair and varied rather than hitscan.

**Comment by Hatterhat** [comment]
dont get hitscanned lol

**Comment by Hatterhat** [comment]
huh wow i really fucked up the branch pretty badly

**Pull request closed without merging**